data_import/templatetags/data_import.py

```diff
@@ -0,0 +1,20 @@
+from django import template
+
+from ..utils import app_name_to_user_data_model
+
+register = template.Library()
+
+
+@register.simple_tag
+def source_is_connected(source, user):
+    """
+    Given 'go_viral', return True or False.
```

Comment:
true, I just updated the description from the tag it was based on--I'll add a better description. :)